### ISSUE TITLE
maisi TSP question

### ISSUE THREAD

**Issue by zpwithme** [Project-MONAI/tutorials#1870]
**Please use MONAI's Discussions tab**
For questions relating to MONAI usage, please do not create an issue.

Instead, use [MONAI's GitHub Discussions tab](https://github.com/Project-MONAI/MONAI/discussions). This can be found on the MONAI repository's main page (not the tutorials) next to Issues and Pull Requests along the top.
Thank you very much for your work, but I see that the training of the VAE is still based on sliding window inference code. Is there any code that uses the TSP method?

**Comment by Can-Zhao** [comment]
Not really...It was trained with sliding window. TSP is used for inference.
